packages/mobile/src/paymentRequest/IncomingPaymentRequestListItem.tsx

```diff
@@ -92,6 +92,11 @@ export default function IncomingPaymentRequestListItem({ id, amount, comment, re
   }
 
   React.useEffect(() => {
+    // Need this to make sure it's only triggered on click
+    if (!isLoading) {
```

Comment:
`isLoading` is true whenever the "send" button is clicked. This was sometimes triggering when lookups were being done as part of other flows

Reply:
wouldnt you want to add this to the deps array?

Reply:
it also looks like is oading is only true when there is an e164 number.

Reply:
I didn't add to the dependency array because I don't actually want that effect to run when `isLoading` changes, only when there are is an update to `secureSendDetail` or `error`. `isLoading` will be `true` whenever those variables change in the vast majority of cases so that check is just to cover edge cases

Reply:
And when there isn't an `e164PhoneNumber` then the lifecycle function is irrelevant because we will immediately navigate to the next screen